### PR TITLE
Fix(web): Fix default Icon size overriding svg attributes

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

**Review by literat — APPROVED** [review]
LGTM

File: packages/web/src/scss/components/Icon/_Icon.scss [hunk 1]
```diff
@@ -1,7 +1,3 @@
-@use 'theme';
-
 .Icon {
     display: inline-block;
```
Comment (adamkudrna) [inline packages/web/src/scss/components/Icon/_Icon.scss:2]: Is this necessary at all? Is there any difference?

**Review by adamkudrna — CHANGES_REQUESTED** [review]
Please mention in `README` that dimensions are mandatory.

File: packages/web/src/scss/components/Icon/_Icon.scss (hunk 1 above)
Comment (crishpeen) [reply packages/web/src/scss/components/Icon/_Icon.scss:2]: TODO consider removing Icon styles at all. As we set up a size on SVG element directly we might not need this.

Comment (adamkudrna) [reply packages/web/src/scss/components/Icon/_Icon.scss:2]: Please remove the CSS in this PR and then I'll see if it's convenient to move all icons elsewhere.

**Review by adamkudrna — CHANGES_REQUESTED** [review]

**Comment by crishpeen** [comment]
Closing in favor of #489

**Pull request closed without merging**